Framework/Crystal/src/SetGoniometer.cpp

```diff
@@ -10,6 +10,7 @@
 #include "MantidAPI/Run.h"
 #include "MantidGeometry/Instrument/Goniometer.h"
 #include "MantidKernel/ListValidator.h"
+#include "MantidKernel/Matrix.h"
```

Comment:
I think this include is not necessary as you've already added it to the header file.

Reply:
Just checked and it compiles and works if you comment out this line
```suggestion
```